ui/store/institutional/institution-background.ts

```diff
@@ -35,6 +35,22 @@ export function showInteractiveReplacementTokenBanner({
     }
   };
 }
+
+export function setTypedMsgInProgress(msgId: string) {
+  return async (dispatch: any) => {
+    dispatch(showLoadingIndication());
+    try {
+      await submitRequestToBackground('setTypedMsgInProgress', [msgId]);
```

Comment:
@brad-decker this `setTypedMsgInProgress` doesn't exist anymore right? Should we use the `signTypedMessage` instead? Or any other method from the [SignController](https://github.com/MetaMask/metamask-extension/blob/develop/app/scripts/controllers/sign.ts)

Reply:
We have another PR to tackle this, in the making now: https://github.com/MetaMask/core/pull/1339
👍🏼